samples/winapi_drivelist/main.c

```diff
@@ -56,11 +58,25 @@ int main(void)
         debugPrint("%s\n", drive);
         while(*drive++);
     }
-    debugPrint("\ndone");
 
-    while(1) {
-        Sleep(2000);
+    debugPrint("\nDone!");
+
+    goto unmount_e;
+
+unmount_e:
+    ret = nxUnmountDrive('E');
+    // If there was an error while unmounting
```

Comment:
This comment adds nothing in my opinion.
The "GetLastError" in combination with `!ret` strongly implies that this is error-checking.